### PR TITLE
js styles for FixedGrid

### PR DESCRIPTION
**Pull request by chrismohr**
## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

File: src/components/FixedGrid/FixedGridRow.styles.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  gutterSize: GutterSize;
+}
+
+export const getClassNames = memoizeFunction((styleProps: BoxStyleProps) => {
+  const { bottomSpacing, gutterSize } = styleProps;
+
+  return mergeStyleSets({
+    root: {
+      display: 'flex',
+      marginBottom: bottomSpacing ? gutterSizes[bottomSpacing] : undefined,
```
Comment (chrismohr) [inline src/components/FixedGrid/FixedGridRow.styles.ts]: When https://github.com/Microsoft/YamUI/pull/380 is merged, we can use `getGutterValue` here instead.

File: src/components/FixedGrid/FixedGridColumn.styles.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { mergeStyleSets } from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+
+export interface FixedGridColumnStyleProps {
+  fixed?: boolean;
+  width?: number;
+  verticalAlign: 'top' | 'middle' | 'bottom';
+}
+
+const alignItems: { [i in 'top' | 'middle' | 'bottom']: 'flex-start' | 'flex-end' | 'center' } = {
```
Comment (swese44) [inline src/components/FixedGrid/FixedGridColumn.styles.ts]: TypeScript is now smart enough map enum values to object keys directly. See here:
https://github.com/Microsoft/YamUI/pull/380/files#diff-633c2868943735bc3166c3960a5bcd29R13

and usage here:
https://github.com/Microsoft/YamUI/pull/380/files#diff-633c2868943735bc3166c3960a5bcd29R29


does this not work for the string literal union types?

File: src/components/FixedGrid/FixedGridRow.tsx [hunk 3]
```diff
@@ -24,27 +26,24 @@ export { GutterSize };
...
-    if (bottomSpacing) {
-      classes.push(`y-fixedGridRow__bottomSpacing-${bottomSpacing}`);
-    }
-    if (className) {
-      classes.push(className);
-    }
-
-    return classes.join(' ');
-  }
+  private childrenWithClassName = (className: string) => {
```
Comment (swese44) [inline src/components/FixedGrid/FixedGridRow.tsx]: this is pretty ugly. can we just move the margin styles to a selector on the parent row?

File: src/components/FixedGrid/FixedGridColumn.styles.ts (hunk 2 above)
Comment (chrismohr) [reply src/components/FixedGrid/FixedGridColumn.styles.ts]: The trouble here was that I need the values to be typed. `alignItems` can't be a string.  Do you know if it's possible to type the values without typing the keys?

File: src/components/FixedGrid/FixedGridRow.tsx (hunk 3 above)
Comment (chrismohr) [reply src/components/FixedGrid/FixedGridRow.tsx]: Good idea.  I'll try.

**Pull request merged**